test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -257,6 +257,32 @@ public async Task GivenResources_WhenSearchedWithIncorrectSummaryParams_ThenExce
             Assert.Equal(HttpStatusCode.BadRequest, ex.StatusCode);
         }
 
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenListOfResources_WhenSearchedWithElements_ThenOnlySpecifiedPropertiesShouldBeReturned()
```

Comment:
Would be good to have some negative/empty/invalid cases too

Reply:
I'm a bit preoccupied with another project. Can add some more tests towards the end of the week

Reply:
I've added negative tests